src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,250 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+import { useListData } from "@react-stately/data";
+import { ItemDropTarget } from "@react-types/shared";
+import { Meta, Story } from "@storybook/react";
+import React, { FC, forwardRef, ReactChild, useState } from "react";
+import { Button, ButtonSize } from "@components/Button";
+import { OrderableList as OrderableListComponent } from "./OrderableList";
+import { merge } from "@utilities/merge";
+import { GridNode } from "@react-types/grid";
+import { DragProperties, OrderableListItem } from "./types";
+import { FocusController, FocusControllerWidth, ItemDragState, OrderableListProps } from ".";
+import { chain } from "@react-aria/utils";
+import { Checklist, ChecklistDirection } from "@components/Checklist";
+import { Textarea } from "@components/Textarea";
+import { ButtonGroup } from "@components/ButtonGroup";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+    title: "Components/Orderable List",
+    component: OrderableListComponent,
+    args: {
+        dragDisabled: false,
+        disableTypeAhead: true,
+    },
+    argTypes: {
+        onMove: { action: "onMove" },
+    },
+} as Meta<OrderableListProps<StoryListItem>>;
+
+export const dragStoryStyles: Record<ItemDragState, string> = {
```

Comment:
You cannot export any other constants inside story files than the story itself. There's an error when trying to open the stories

Reply:
fixed